docs/patterns/distribute.rst

```diff
@@ -16,23 +15,23 @@ make larger applications easier to distribute:
   Python installation.  This makes it possible to query information
   provided by one package from another package.  The best known feature of
   this system is the entry point support which allows one package to
-  declare an "entry point" another package can hook into to extend the
+  declare an "entry point" that another package can hook into to extend the
   other package.
-- **installation manager**: :command:`easy_install`, which comes with distribute
-  can install other libraries for you.  You can also use `pip`_ which
+- **installation manager**: :command:`easy_install`, which comes with setuptools
```

Comment:
I just pushed a change that removes the references to easy_install and adds a note about installation of setuptools and pip.